doc/source/whatsnew/v0.22.0.txt

```diff
@@ -167,5 +167,5 @@ Other
 ^^^^^
 
 - Improved error message when attempting to use a Python keyword as an identifier in a numexpr query (:issue:`18221`)
--
+- Added teardown functions in asv benchmarks for hdfstore, io and packers benches.
```

Comment:
@jreback @jorisvandenbossche : Do you we do `whatsnew` for these types of changes?

Reply:
Typically not. There are a bit two conflicting aspects of the whatsnew file: informing users about changes/fixes (and then this is not needed in there as no user impact) and valuing contributors (by having their PR listed in the whathsnew note).

Reply:
Okay, just wanted to double-check.  Given that is not user-facing, @dmanikowski-reef , could you revert your addition to the `whatsnew` ?

@jorisvandenbossche : Not sure if they have to be conflicting (e.g. could we list the PR's accepted between releases at the end of the `whatsnew`?) - however, can always continue this outside of this PR if need be.